src/App.tsx

```diff
@@ -1,61 +1,122 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+const todosWithUsers = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [currentTitle, setCurrentTitle] = useState('');
+  const [currentUserId, setCurrentUserId] = useState(0);
+
+  const [hasNameError, setHasNameError] = useState(false);
```

Comment:
Should write the name of the state a little bit more appropriately
```suggestion
  const [hasTitleError, setHasTitleError] = useState(false);
```